example/src/main/java/de/example/nn/MultilayerPerceptronSeabornExample.java

```diff
@@ -0,0 +1,50 @@
+package de.example.nn;
+
+
+import de.edux.api.Classifier;
+import de.edux.functions.activation.ActivationFunction;
+import de.edux.functions.initialization.Initialization;
+import de.edux.functions.loss.LossFunction;
+import de.edux.ml.knn.KnnClassifier;
+import de.edux.ml.nn.config.NetworkConfiguration;
+import de.edux.ml.nn.network.MultilayerPerceptron;
+import de.edux.ml.nn.network.api.Dataset;
+import de.example.data.seaborn.Penguin;
+import de.example.data.seaborn.SeabornDataProcessor;
+import de.example.data.seaborn.SeabornProvider;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Knn - K nearest neighbors
+ * Dataset: Seaborn Penguins
+ */
+public class MultilayerPerceptronExample2 {
```

Comment:
Rename this to MultilayerPerceptronSeabornExample

Also Rename the old MultilayerPerceptronExample class to MultilayerPerceptronIrisExample